hca-to-scea/helpers/utils.py

```diff
@@ -0,0 +1,260 @@
+#
```

Comment:
`utils` is a name that suggests this file contains a number of things that are not related to one another. It means it would have many reasons to change, as opposed to a single one.
I see here "pure" calculation related functions, as well as ones that seem to be related to our domain.
In addition, the file is very long, which also suggests there is too much going on in this file.